src/components/Home/Home.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import API from '../../Services/axios';
+import { Headers } from '../Header'
+export const Home = () => {
+  API.get(`users/dummy/`)
```

Comment:
why are we doing async actions directly in the component? Why not using useEffect?